core/reporter-api/src/main/java/org/arquillian/reporter/api/model/report/AbstractReport.java

```diff
@@ -1,14 +1,14 @@
 package org.arquillian.reporter.api.model.report;
 
-import java.util.ArrayList;
-import java.util.List;
-
+import org.arquillian.reporter.api.builder.report.ReportBuilder;
 import org.arquillian.reporter.api.model.StringKey;
 import org.arquillian.reporter.api.model.UnknownStringKey;
 import org.arquillian.reporter.api.model.entry.Entry;
-import org.arquillian.reporter.api.builder.report.ReportBuilder;
 import org.arquillian.reporter.api.utils.Validate;
 
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
That's true :) I thought more of

For imports as well as curly braces and so on we have to look for some other approach. maybe sharing IDE specific settings will be enough